general_documentation/dunning_kruger_effect.md

```diff
@@ -0,0 +1,41 @@
+
+### Underestimation by Juniors
+Have you ever thought a task would be easy only for it to become an insurmountable one? A new-hire may underestimate the difficulty of a similar task because of the "ease" by which she completed a previously similar task. It happens ALL THE TIME! It is very true that "You do not know what you do not know". The new-hire may have been unaware of the challenges that lay ahead, challenges that a more senior hire would have seen. The person might even be a below-average worker who just "got lucky" on a few tasks. Unfortunately, the person became overconfident -- and yes this is the Dunning Kruger effect at its finest!  
+
+### Estimation by Everyone
+Charles Darwin, in his book "The Descent of Man" [^darwin], wrote “Ignorance more frequently begets confidence than does knowledge”. In short, unfamiliarity in a topic can cause a person to be confidence in misinformation.
+
+****I'm a professional race car driver!****  
+On a small scale, individuals can make erroneous decisions based on an estimation of their own abilities, which could be based on a fallacy. For example, a 1986 comparative study on US drivers[^drivers] concluded that 80% of people considered themselves as "above average drivers". This is basically saying that 80% of US drivers considered themselves better than remaining 50% of the population.  That is also like slicing a pizza into a hundred (100) slices and saying eighty (80) of the slices are larger than the remaining fifty (50). I am sorry, come again? Is this why so many US drivers attempt to weave in-and-out of traffic as if they received nascar level training?  
+
+****I'm an indispensable employee!****  
+Another study[^developers] founded that 42% of employees in a software engineering company assessed their own performance as the top 5%. These employees overestimate their own abilities and may end up looking down on others either intentionally or unintentionally. This can rob team members of valuable opportunities to learn, grow, and develop as IT professionals.
+
+### Conclusion
+The Dunning Kruger Effect is a cause-and-effect outcome based on a person's use of misinformation. This misinformation allows for a catch-22 scenario because it can happen without awareness or knowledge. It is important for IT professionals to understand that it is expected for them to make the erroneous decision from time-to-time. And it is expected for IT professionals to sometimes over/underestimate by a lot. The goal is to never let Dunning-Kruger Effect influence your perception of yourself or others!
```

Comment:
- replace "use of misinformation" with "misunderstanding" (misinformation is more so like propaganda)
- replace "This misinformation allows" with "These misgivings allow"
- What's the other side of the catch-22? It can happen unknowingly, but what's the other side of this?
- replace "without awareness or knowledge" with "unknowingly"
- no hyphens in "time to time"
- format like this: "over- or underestimate"
- insert "the" before "Dunning-Kruger Effect"